app/models/account.rb

```diff
@@ -230,8 +230,8 @@ def fields_attributes=(attributes)
     fields     = []
     old_fields = self[:fields] || []
 
-    if attributes.is_a?(Hash)
-      attributes.each_value do |attr|
+    if attributes.is_a?(Array)
```

Comment:
This change isn't right either, because we get a hash from a standard form submission. So it can be a hash or an array.

Reply:
Change to 
`Array.wrap(attributes).each do |attr|`

We wrap the array around a single element to allow iteration.

Reply:
@Gargron You can check the latest commit